job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/S3BucketPublisherContext.groovy

```diff
@@ -41,6 +41,8 @@ class S3BucketPublisherContext extends AbstractContext {
             noUploadOnFailure(context.noUploadOnFailure)
             uploadFromSlave(context.uploadFromSlave)
             managedArtifacts(context.managedArtifacts)
+            useServerSideEncryption(context.useServerSideEncryption)
```

Comment:
To maintain compatibility, these element must only be generated when a matching version of the S3 plugin is installed, see [PublisherContext.groovy#L310](https://github.com/jenkinsci/job-dsl-plugin/blob/master/job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy#L310) for an example.